cmake/cmake_config.hpp.in

```diff
@@ -0,0 +1,15 @@
+std::cout << "CMAKE_CXX_FLAGS=" << "@CMAKE_CXX_FLAGS@" << std::endl;
+std::cout << "CMAKE_C_FLAGS=" << "@CMAKE_C_FLAGS@" << std::endl;
+std::cout << "CMAKE_BUILD_TYPE=" << "@CMAKE_BUILD_TYPE@" << std::endl;
+
+std::cout << "OOMPH_UCX_USE_PMI=" << "@OOMPH_UCX_USE_PMI@" << std::endl;
+std::cout << "OOMPH_UCX_USE_SPIN_LOCK=" << "@OOMPH_UCX_USE_SPIN_LOCK@" << std::endl;
+std::cout << "OOMPH_USE_FAST_PIMPL=" << "@OOMPH_USE_FAST_PIMPL@" << std::endl;
+std::cout << "OOMPH_UCX_USE_MULTIPLE_ENDPOINTS=" << "@OOMPH_UCX_USE_MULTIPLE_ENDPOINTS@" << std::endl;
```

Comment:
can we leave this one out until we have this option?

Reply:
oh, sure.